bridge/irc_manager.go

```diff
@@ -20,14 +20,16 @@ var indev = false
 type IRCManager struct {
 	ircConnections map[string]*ircConnection
 
-	bridge *Bridge
+	bridge     *Bridge
+	nickLength int
 }
 
 // NewIRCManager creates a new IRCManager
-func newIRCManager(bridge *Bridge) *IRCManager {
+func newIRCManager(bridge *Bridge, nickLength int) *IRCManager {
 	return &IRCManager{
 		ircConnections: make(map[string]*ircConnection),
 		bridge:         bridge,
+		nickLength:     nickLength,
```

Comment:
This nickLength stuff can be removed now

Reply:
Removed